kubernetes/assets/recommended_monitors/monitor_pod_memory_limit.json

```diff
@@ -0,0 +1,28 @@
+{
+	"name": "[kubernetes] Monitor Kubernetes Pod Memory Limit Reached",
+	"type": "query alert",
+	"query": "avg(last_5m):exclude_null(avg:kubernetes.memory.rss{*} by {pod_name,kube_service}) / avg:kubernetes.memory.limits{*} by {pod_name,kube_service} * 100 > 95",
```

Comment:
check whether kube_service is a standard metric available in the app.

Reply:
@sanjuashok kube_service is a standard tag:  
https://docs.datadoghq.com/infrastructure/livecontainers/?tab=kubernetes#tagging